src/bin/vip-wp.js

```diff
@@ -280,7 +329,7 @@ commandWrapper( {
 
 				if ( ! isSubShell ) {
 					subShellRl.close();
-					process.exit();
+					shutdownHandler( false );
```

Comment:
I guess leaving this `process.exit();` is better than wrapping it with `shutdownHandler( false );`. Especially if we aren't adding anything in `shutdownHandler` other than calling `process.exit();` inside (which is the current case when passing `false`)